src/odbc_connection.cpp

```diff
@@ -2127,7 +2166,11 @@ class TablesAsyncWorker : public ODBCAsyncWorker {
       data(data) {}
 
     ~TablesAsyncWorker() {
-      delete data;
+      if (data != NULL)
+      {
+        delete data;
+        data = NULL;
+      }
```

Comment:
These changes are pointless since it's safe to delete a NULL pointer (`delete` has this `if` check already): https://stackoverflow.com/questions/4190703/is-it-safe-to-delete-a-null-pointer